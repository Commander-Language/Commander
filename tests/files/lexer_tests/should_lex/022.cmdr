// Test that if, else if, and else statements lex properly
if(true){
} else if(true){
} else {
}
