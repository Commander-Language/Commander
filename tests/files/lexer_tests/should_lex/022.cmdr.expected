IF 'if' 2:1
LPAREN '(' 2:3
TRUE 'true' 2:4
RPAREN ')' 2:8
LCURLY '{' 2:9
RCURLY '}' 3:1
ELSE 'else' 3:3
IF 'if' 3:8
LPAREN '(' 3:10
TRUE 'true' 3:11
RPAREN ')' 3:15
LCURLY '{' 3:16
RCURLY '}' 4:1
ELSE 'else' 4:3
LCURLY '{' 4:8
RCURLY '}' 5:1
END_OF_FILE 6:1
